tests/data/dbAuth/good_test.yaml

```diff
@@ -0,0 +1,40 @@
+[
+{
+        url: "postgresql://user@host.example.com:5432/my_database",
```

Comment:
I don't like this YAML form because of all the white space and extra brackets everywhere.  Can't we use:
```yaml
- password: test1
  url: postgresql://user@host.example.com:5432/my_database
- password: test2
  url: postgresql://user@host.example.com/my_database
- password: test3
  url: postgresql://host.example.com:5432/my_database
  username: user
- password: test4
  url: postgresql://host.example.com/my_database
  username: user
- password: test5
  url: postgresql://*.example.com/my_*
  username: user
- password: test6
  url: postgresql://*.example.com/
  username: user
- password: test7
  url: postgresql://*.example.com
  username: user
- password: test8
  url: postgresql://*
  username: user
```?

Reply:
Sure.  Although password-first seems strange.

Reply:
Yes, I didn't notice. That's the order that came out when I dumped the list that I had read from the initial yaml. It sorted the keys.